app/gui/src/model/module/plain.rs

```diff
@@ -480,4 +480,29 @@ mod test {
             .unwrap();
         assert_eq!(Some(new_pos), module.node_metadata(id).unwrap().position);
     }
+
+    #[test]
+    fn test_metadata_stability() {
+        // TODO [mwu]
+        //  This tests makes sure that the program included below properly round-trips through
+        //  AST <-> Text conversions. This is specifically for testing workaround for the bug
+        //  https://github.com/enso-org/enso/issues/6718. After proper fix is implemented, this
+        //  test should be removed or accordingly modified (remove the module ast id).
+        const PROGRAM: &str = r#"from Standard.Base import all
+from Standard.Table import all
+from Standard.Database import all
+from Standard.AWS import all
+import Standard.Visualization
+
+main =
+    operator1 = "Press TAB key to create a new node"
+
```

Comment:
Can we in such cases maintain the code indentation? We can before passing this to engine just cut the left indentation to the common one, making this file looks nicer. What do you think of this idea?

Reply:
There are pros and cons to both approaches. Personally, I prefer to keep the indentation as-is, because it makes it easier to:
* copy-paste code as-is to test it;
* easily update code without reformatting.
As this is used only in tests (which are generally not long and separated from other code), the irregular indention doesn't bother me much. 

But this is not a strong opinion and generally matter of taste.

Some compromise might be to the code to separate files, and then use `include` to load them. This way, the code is still separated, but the indentation is regular. On the other hand, it becomes impossible to understand the code at a glance, because you have to jump between files.

At this time I'd keep it as-is, as this convention is also used in many other tests. If we decide we want to use other style, I'd rather have it a separate PR that changes all tests at once, so that we don't have a mix of styles.

Reply:
I'm ok with leaving it this way! Thanks for the explanation!